omegaconf/basecontainer.py

```diff
@@ -91,12 +92,14 @@ def __repr__(self) -> str:
     def __getstate__(self) -> Dict[str, Any]:
         dict_ = copy.copy(self.__dict__)
```

Comment:
for clarity, can you rename this to `dict_copy` ?

Reply:
@omry done.